content/docs/04-clusters/03-edge/02-edge-configuration/30-cloud-init.md

```diff
@@ -59,6 +59,63 @@ You may ask yourself where to use cloud-init stages, as both the Edge Installer
+
+  Once the edge host is prepared with the initial installation, you remove the USB flash drive and ship your devices to the site for installation. This step is also called the *installer handoff* step. Refer to the [Prepare Edge Host
+](/clusters/edge/site-deployment/stage#prepareedgehost) guide to learn more about driving the installer handoff step.
+
+
+- **Site Installation Step** - In the site installation step, you use supplementary user data to apply site-specific configurations to the edge devices. In this step, you create another USB flash drive using an ISO that contains your additional user data. You use the USB flash drive to install the site-specific configurations. The entire USB flash drive data is copied to the edge host during the installation unless you follow the specific naming convention for your user data stages as described below. 
+
+  The [Perform Site Install](/clusters/edge/site-deployment/site-installation) guide describes the site installation process in detail. The site installation step is optional. You can refer to the [Multiple User Data Use Case](/clusters/edge/edgeforge-workflow/prepare-user-data#multipleuserdatausecase) guide to learn about the use cases for the site installation step.
+
+
+In both steps mentioned above,  the Edge Installer copies the user data from the USB flash drive to the **/run/stylus/userdata** file or the **/oem/userdata** file on the edge hosts. Suppose you want to keep some user data stages off the edge hosts. In such scenarios, we recommend applying the user data during the site installation step and using the specific naming convention for your user data stages, as described in the [Sensitive Information During the Site Installation](#sensitiveinformationduringthesiteinstallation) section below.
+<br />
+
+## Sensitive Information in the Installer Handoff
+
+<br />
+<WarningBox>
+
+We do not recommend using sensitive information during the installer handoff step. Instead, you must consider using sensitive information during the site installation step.
+
+</WarningBox>
+<br />
+
+In the installer handoff step, the Edge Installer copies and persists *all* your user data stages into the configuration files on the edge hosts. Copying sensitive information to the edge hosts may pose security risks. Therefore, we do not recommend using sensitive information during the installer handoff step. Instead, you must consider using sensitive information during the site installation step.
+
+
+Suppose you consciously want to pass some sensitive information in the user data during the installer handoff step. In that case, you must specify `powerOff: true` in the install section of user data to ensure the devices are shut down when you ship them to the site. 
```

Comment:
```suggestion
```

This is irrelevant to the topic. All it's stating is to power off the device. That has nothing to do with the behavior of persisting sensitive data.